src/defrag.c

```diff
@@ -672,7 +672,7 @@ int scanLaterStreamListpacks(robj *ob, unsigned long *cursor, monotime endtime)
         raxSeek(&ri,"^",NULL,0);
     } else {
         /* if cursor is non-zero, we seek to the static 'last' */
-        if (!raxSeek(&ri,">", last, sizeof(last))) {
+        if (!raxSeek(&ri,">=", last, sizeof(last))) {
```

Comment:
maybe rename `last` to `next`?

Reply:
done with [3923c14 (#13847)](https://github.com/redis/redis/pull/13847/commits/3923c14479ec1a3e7e1f9aa2038a4d84a03bab4a)